react-select/react-select-tests.tsx

```diff
@@ -61,6 +63,30 @@ const CustomValue = React.createClass({
     }
 });
 
+const filterOptions = (options: Array<Option>, filter: string, values: Array<Option>) => {
```

Comment:
It's the temporary solution for allow create (https://github.com/JedWatson/react-select/wiki#allow-create-temporary-solution). I'll open up a follow-up PR to follow your suggestions.